docs/proposals/release-notes.md

```diff
@@ -115,14 +115,15 @@ hundreds of entries.  The goal is to highlight the major changes for a release.
 
 The munger/bot option fits most cleanly into the existing workflow.
 
-The design will include:
+All accounting is managed on the master branch PR only.
```

Comment:
@zmerlynn and I noticed today that this isn't yet true. I'm looking forward to the munger being fixed to follow this process.